packages/react-hooks/src/types.ts

```diff
@@ -384,5 +385,6 @@ export const CoreTimeConsts = {
 };
 
 export const CoreTimeChainConsts = {
```

Comment:
I would argue it's probably better for these `consts` to not live within the `types.ts` file since they are part of the transpiled JS output.

Reply:
good point! Will create constants file for them. I am sharing them between page-coretime and page-broker packages.